README.md

```diff
@@ -16,25 +16,39 @@ Or
 
 
 ## Run with Docker
-
+- If you are on your local machine, we recommend that you run this container using Docker.  
+- We have tested running this container using Docker on local mac machines and also on a Pawsey Nimbus instance.
+  
+### On a local (mac) machine
 To run this container:
-
 - Start docker
 - Mount a directory housing your data (specify which paths you need to mount), and run the RStudio server instance using the following command: 
```

Comment:
How do I do that?

Reply:
Actually this is part of the RStudio server command .. I have now rephrased the sentence and explained the mounting after the command